Source/Core/Core/HW/VideoInterface.h

```diff
@@ -454,7 +454,7 @@ class VideoInterfaceManager
   u32 m_even_field_last_hl = 0;   // index last halfline of the even field
   u32 m_odd_field_last_hl = 0;    // index last halfline of the odd field
 
-  float m_config_vi_oc_factor = 0.0f;
+  float m_config_vi_oc_factor = 1.0f;
```

Comment:
Maybe also change it for the regular overclock, which may not be UB now, but could be later?

https://github.com/dolphin-emu/dolphin/blob/2b7faeb920da31edb11f01c9a0473f86e4023100/Source/Core/Core/CoreTiming.h#L206

Reply:
Good idea, done.